simple/test.js

```diff
@@ -3830,7 +3832,8 @@ exports.test = function (global) {
       if (global.outputJson) {
         fs.writeFileSync("results.json", JSON.stringify(result));
       }
+      return global.returnValue;
```

Comment:
I don't see why we need this variable, it is only used here. I would prefer just returning 0 here.